project/src/store/reducer/data/selectors.ts

```diff
@@ -1,10 +1,19 @@
 import {NameSpace, State} from '../root-reducer';
 import {Offer, Review} from '../../../types/types';
 import {RequestStatus} from './data-reducer';
+import { getCurrentSort, getSelectedCity } from '../app/selectors';
+import { getOffersInCity, Sort } from '../../../utils';
+import { createSelector } from 'reselect';
```

Comment:
Согласно [документации](https://github.com/reduxjs/reselect) можно было бы импортировать эту функцию и из readux toolkit. 
```suggestion
import { createSelector } from '@reduxjs/toolkit';
```